linera-core/src/chain_worker/state/mod.rs

```diff
@@ -381,9 +381,16 @@ where
     }
 
     /// Adds any newly created chains to the set of `tracked_chains`.
-    fn track_newly_created_chains(&self, block: &ExecutedBlock) {
+    fn track_newly_created_chains(&self, executed_block: &ExecutedBlock) {
         if let Some(tracked_chains) = self.tracked_chains.as_ref() {
-            let messages = block.messages().iter().flatten();
+            if !tracked_chains
```

Comment:
Can you add a comment here? Or in the brackets, just before the `return`, explaining why we do that?

Reply:
I extended it in [9aebea0](https://github.com/linera-io/linera-protocol/pull/2737/commits/9aebea05d85f844e28d2f0cfa2f2bd81b97309fb).

(For the longer-term plans for tracked/followed chains, take a look at LIP-21.)